webnode/config/webpack.config.dev.js

```diff
@@ -1,140 +0,0 @@
-              eslintPath: require.resolve("eslint")
-            },
-            loader: require.resolve("eslint-loader")
-          }
-        ],
-        include: paths.appSrc
-      },
-      {
-        // "oneOf" will traverse all following loaders until one will
-        // match the requirements. When no loader matches it will fall
-        // back to the "file" loader at the end of the loader list.
-        oneOf: [
-          {
-            test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
-            loader: require.resolve("url-loader"),
-            options: {
-              limit: 10000,
-              name: "static/media/[name].[ext]"
-            }
-          },
-          {
-            test: /\.(js|jsx|mjs)$/,
-            include: paths.appSrc,
-            exclude: /worker\.js$/,
-            loader: require.resolve("babel-loader"),
-            options: {
-              cacheDirectory: true,
-              presets: ["react-app"],
-              plugins: ["transform-object-rest-spread"]
-            }
-          },
-          {
-            test: /\.css$/,
-            use: [
-              require.resolve("style-loader"),
-              {
-                loader: require.resolve("css-loader"),
-                options: {
-                  importLoaders: 1
-                }
-              },
-              {
-                loader: require.resolve("postcss-loader"),
-                options: {
-                  ident: "postcss",
-                  plugins: () => [
-                    require("postcss-flexbugs-fixes"),
-                    autoprefixer({
-                      browsers: [
-                        ">1%",
-                        "last 4 versions",
-                        "Firefox ESR",
-                        "not ie < 9"
-                      ],
-                      flexbox: "no-2009"
-                    })
-                  ]
-                }
-              }
-            ]
-          },
-          {
-            test: /worker\.js$/,
-            use: {
-              loader: "worker-loader",
-              options: {
-                name: "[name].[ext]",
-                publicPath: "src/redux/workers/"
-              }
-            }
-          },
-          {
-            exclude: [/\.(js|jsx|mjs)$/, /\.html$/, /\.json$/],
-            loader: require.resolve("file-loader"),
-            options: {
-              name: "static/media/[name].[ext]"
-            }
-          }
-        ]
-      }
-      // ** STOP ** Are you adding a new loader?
-      // Make sure to add the new loader(s) before the "file" loader.
-    ]
-  },
-  plugins: [
-    new InterpolateHtmlPlugin(env.raw),
-    new HtmlWebpackPlugin({
-      inject: true,
-      template: paths.appHtml
-    }),
-    new webpack.EnvironmentPlugin({
-      NODE_ENV: "development",
-      DEBUG: true
-    }),
-    new ScriptAttrHtmlWebpackPlugin({
-      attributes: {
-        "eth-address": "0xD1833A50f411432aD38E8374df8Cfff79e743788"
```

Comment:
Was removing this file intentional?  When we build locally, will we still have this eth address?

Reply:
the webpack dev config and prod config were merged into one. its now webpack.config